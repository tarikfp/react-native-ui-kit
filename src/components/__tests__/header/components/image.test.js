import * as React from "react";
import renderer from "react-test-renderer";
import HeaderImage from "../../../header/components/image";

describe("tests header image component", () => {
  it("renders header image with source", () => {
    const tree = renderer
      .create(
        <HeaderImage
          style={{ width: 50, height: 50 }}
          source={{
            uri: "https://reactjs.org/logo-og.png"
          }}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
