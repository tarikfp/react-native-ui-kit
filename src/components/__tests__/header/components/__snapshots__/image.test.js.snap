// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests header image component renders header image with source 1`] = `
<Image
  source={
    Object {
      "uri": "https://reactjs.org/logo-og.png",
    }
  }
  style={
    Object {
      "height": 50,
      "width": 50,
    }
  }
/>
`;
